docs/source/config_files.rst

```diff
@@ -32,6 +32,8 @@ Below is a list of supported configuration files (roughly in the order of build
 ``environment.yml`` is the standard configuration file used by `conda <https://conda.io>`_
 that lets you install any kind of package,
 including Python, R, and C/C++ packages.
+``repo2docker`` does not use ``environment.yml`` to create and activate a new conda environment.
+Rather, it updates a base conda environment with the packages listed in ``environment.yml``.
```

Comment:
```suggestion
Rather, it updates a base conda environment with the packages listed in ``environment.yml``.
This means that the environment will always have the same name and not follow what is
specified in ``environment.yml``.
```

What do you think of adding something like this to explain to people when this would be relevant or a  point of confusion?